tests/libs/benchmark/src/MemoryInfo.cpp

```diff
@@ -31,7 +31,7 @@ const std::string proc_status_path("/proc/self/status");
 const std::string gpu_memory_path("/sys/kernel/debug/mali0/gpu_memory");
 const std::string proc_smaps_path("/proc/self/smaps");
 
-bool isStrNumber(const std::string &s)
+[[maybe_unused]] bool isStrNumber(const std::string &s)
```

Comment:
`maybe_unused` seems to be c++ attribute that makes build looser rather than stricter.

Reply:
`isStrNumber` is used on debug build only by `assert(...)`. Please let me know better way to avoid build failure by unused function on release build.

Reply:
Does `isStrNumber` cause build failure when `ENABLE_STRICT_BUILD=ON`?